bigtable-hbase-replication/src/main/java/com/google/cloud/bigtable/hbase/replication/CloudBigtableReplicator.java

```diff
@@ -195,34 +195,21 @@ public UUID getPeerUUID() {
     return UUID.nameUUIDFromBytes("Cloud-bigtable".getBytes(StandardCharsets.UTF_8));
   }
 
-  public boolean replicate(List<WAL.Entry> walsToReplicate) {
+  public boolean replicate(Map<String, List<BigtableWALEntry>> walEntriesByTable) {
```

Comment:
Interesting, how are you making it work? The BigtableWALEntryImpl classes don't extend/implement the BigtableWALEntry, how does it work?